jmeter/0.3.0/README.md

```diff
@@ -0,0 +1,198 @@
+# JMeter Job-Executor Integration
+
+This integration shows you how to leverage [Job-executor-service](https://github.com/keptn-contrib/job-executor-service) for running load tests using `JMeter` on your Kubernetes cluster.
+
+**Please note, that the instructions provided here enable you to run and configure your Kubernetes deployments using the job-executor-service. They are in no way intended to be a complete reference of Kubernetes, nor JMeter, nor Keptn.**
+
+## Installation and Configuration
+
+### Step 1: Install the Job-Executor in your cluster
+
+Install [Job-Executor](https://github.com/keptn-contrib/job-executor-service) in a version compatible with your Keptn installation (see [GitHub Releases page](https://github.com/keptn-contrib/job-executor-service/releases), e.g., version 0.2.5 is compatible with Keptn 0.17.x), and make sure it is subscribed to the following Keptn Cloud Events:
+
+* `sh.keptn.event.test.triggered`
+
+This can verified in Keptn Bridge -> Project -> Settings -> Integrations -> job-executor-service.
+
+**Example Installation Instructions**
+ Please update `JES_VERSION` and `JES_NAMESPACE` in the example below according to your needs.
+
+```bash
+TASK_SUBSCRIPTION='sh.keptn.event.test.triggered'
+JES_VERSION=0.2.5
+JES_NAMESPACE=keptn-jes
+
+helm upgrade --install --create-namespace -n ${JES_NAMESPACE} \
+  job-executor-service https://github.com/keptn-contrib/job-executor-service/releases/download/${JES_VERSION}/job-executor-service-${JES_VERSION}.tgz \
+  --set remoteControlPlane.autoDetect.enabled="true",remoteControlPlane.topicSubscription="${TASK_SUBSCRIPTION}",remoteControlPlane.api.token="",remoteControlPlane.api.hostname="",remoteControlPlane.api.protocol=""
+ ```
+
+**If you have installed jmeter-service, uninstall it**
+
+```bash
+helm uninstall jmeter-service -n keptn
+```
+
+### Step 2: Build your own JMeter Docker image
+
+Since there are no official JMeter Docker images, we recommend to build your own (and customize it). Here is a Dockerfile with a basic JMeter installation, feel free to adapt!
```

Comment:
```suggestion
Since there are no official JMeter Docker images, we recommend that you  build and customize your own. Here is a Dockerfile with a basic JMeter installation, feel free to adapt it!
```